lua/lspinstall/servers/java.lua

```diff
@@ -0,0 +1,48 @@
+local config = require'lspconfig'.jdtls.document_config
+require'lspconfig/configs'.jdtls = nil -- important, immediately unset the loaded config again
+
+config.default_config.cmd[1] = {"./jdtls.sh", vim.fn.expand('$HOME/.local/share/nvim/lspinstall/java/workspace/') .. vim.fn.fnamemodify(vim.fn.getcwd(), ':p:h:t')}
+
+return vim.tbl_extend('error', config, {
+  install_script = [[
+  curl -L -o - http://download.eclipse.org/jdtls/snapshots/jdt-language-server-latest.tar.gz | tar xvz
+  curl -L -o lombok.jar https://projectlombok.org/downloads/lombok.jar
+cat << EOF > jdtls.sh
+#!/usr/bin/env bash
+OS=$(uname)
+
+case \$OS in
+  Linux)
+    CONFIG="$HOME/.local/share/nvim/lspinstall/java/config_linux"
+    ;;
+  Darwin)
+    CONFIG="$HOME/.local/share/nvim/lspinstall/java/config_mac"
+    ;;
+esac
+
+WORKSPACE="$(pwd)/workspace"
+JAR="$(find $HOME/.local/share/nvim/lspinstall/java/plugins/ -name 'org.eclipse.equinox.launcher_*.jar')"
```

Comment:
`$HOME` not escaped. But instead you should use `\$(dirname \$0)/java/plugins/` anyways.

Reply:
Also using `find` is not necessary here, shell name expansion works with `*`